css/documentation.css

```diff
@@ -346,3 +346,18 @@ color: #F04124;
 }
 
 /* end search */
+
+/* begin reading_time */
+
+span.reading-time {
+  font-weight: bold;
```

Comment:
@mstanleyjones  I think it looks better with bold off. Other than that, I say we pull this is and work with design on better styling in the general effort.